shared/src/packages/chat/libs/enums/open-ai-image-size.enum.ts

```diff
@@ -0,0 +1,7 @@
+const OpenAiImageSize = {
```

Comment:
everything related to openAi should be in backend, frontend doesn't know about this implementation